apps/examples/ble_perfs/README.txt

```diff
@@ -0,0 +1,8 @@
+examples/ble_rmc
+^^^^^^^^^^^^^^
+
+  This is BLE remote controller example.
+
+  Configs (see the details on Kconfig):
+  * CONFIG_EXAMPLES_BLE_RMC
```

Comment:
here as well is it CONFIG_EXAMPLES_BLE_PERFS?

Reply:
done